entries/o/one.com.json

```
@@ -1,13 +1,13 @@
 {
   "One.com": {
     "domain": "one.com",
-    "url": "https://www.one.com/",
-    "contact": {
-      "facebook": "onecom",
-      "twitter": "onecom"
-    },
+    "tfa": [
+      "totp",
+      "custom-software"
+    ],
```

The custom software app needs to be specified:
```suggestion
    ],
    "custom-software": [
      "one.com companion"
    ],
```